x12-edi/2.5/antora.yml

```diff
@@ -1,6 +1,6 @@
 name: x12-edi-connector
-version: '2.1'
-display_version: 2.1 (Mule 4)
+version: '2.5'
```

Comment:
Wow, did the version jump from .1 to .5?

Reply:
I"ts a long story.